README.md

```diff
@@ -1,21 +1,21 @@
 # California Jobs Schema
-###**Exploring ALL California Job Classification Schemas!**
+### **Exploring ALL California Job Classification Schemas!**
 
 Exploratory project intended to make CalHR job classification data useful in parsed form; this will enable enhanced sorting, in-depth analysis, ladder analysis, cross-referencing, clustering and more.  
 
 All collaboration is welcome, specifically re: pdf conversion to txt (see known issues), front end, sqlite db (tiny, ~10,000 rows), content copy
 
-####**Currently:**
+#### **Currently:**
 * Uses wget to download the most recent version of the alphabetic jobs classification schema
 * Python parsing of all fields using string manipulation and export to txt, csv, etc.
 
-####**Known Issues:**
+#### **Known Issues:**
 * Requires consistent, automated way to convert PDF to raw text
  * Have considered xpdf, pdftotxt, ghostscript, ps2ascii; all work.. but text recognition formats are inconsistent, even without considering OCR
  * This is still a manual process as of this posting
```

Comment:
cant do the suggestion myself but these nested bullets need 2 spaced to indent properly in the UI long since deleted my fork of for this patch PR, please add the suggestion to add addition space before list item here